src/main/java/study/Operations.java

```diff
@@ -0,0 +1,32 @@
+package study;
+
+import java.util.function.BiFunction;
+
+public enum Operations {
+    PLUS("+",(a,b)->a+b),
+    MINUS("-",(a,b)->a-b),
+    MULTIPLY("*",(a,b)->a*b),
+    DIVIDE("/",(a,b)->a/b);
```

Comment:
```suggestion
    PLUS("+", (a, b) -> a + b),
```
가독성을 조금 더 신경쓰면 좋을 것 같습니다~